CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 
 ### Removed
 
+- Individual etcd member's services were removed. Use hostname and subdomain of etcd pod instead.
```

Comment:
yes. but by `<pod-hostname>.<pod-subdomain>.<namespace>.svc.cluster.local`

i think the pod is also accessible through x.pod.cluster.local.